plugins/woocommerce-admin/client/layout/controller.js

```diff
@@ -290,6 +294,18 @@ export const getPages = () => {
 		} );
 	}
 
+	if ( window.wcAdminFeatures[ 'customize-store' ] ) {
+		pages.push( {
```

Comment:
I'm thinking maybe we should considering put `customize-store` in the task list folder and registering it there since it's an onboarding task. What do you think? I don’t have a strong opinion, just wanted to bring it up for discussion

Reply:
I looked into this a bit but it seems like task list views are not meant to be fullscreen so some of the elements and styles bleed in and will need to be overwritten with more specific styles. 

<img width="1778" alt="image" src="https://github.com/woocommerce/woocommerce/assets/27843274/a3c6ef9f-4a80-4221-8a6f-ce2c052408ad">